.openpublishing.redirection.json

```
@@ -1358,6 +1358,31 @@
             "source_path": "docs/framework/install/net-framework-3-5-on-windows-8-plus.md",
             "redirect_url": "/dotnet/framework/install/dotnet-35-windows-10"
         },
+        {
+            "source_path" : "docs/framework/interop/com-wrappers.md",
+            "redirect_url": "/dotnet/standard/native-interop/com-wrappers",
+            "redirect_document_id": true
+        },
+        {
+            "source_path" : "docs/framework/interop/com-callable-wrapper.md",
+            "redirect_url": "/dotnet/standard/native-interop/com-callable-wrapper",
+            "redirect_document_id": true
+        },
+        {
+            "source_path" : "docs/framework/interop/runtime-callable-wrapper.md",
+            "redirect_url": "/dotnet/standard/native-interop/runtime-callable-wrapper",
+            "redirect_document_id": true
+        },
+        {
+            "source_path" : "docs/framework/interop/applying-interop-attributes.md",
+            "redirect_url": "/dotnet/standard/native-interop/applying-interop-attributes",
```

I'd the opportunity to fix some SEO and naming issues here as well

```suggestion
            "redirect_url": "/dotnet/standard/native-interop/apply-attributes",
```